app/endpoints/groups.py

```diff
@@ -0,0 +1,62 @@
+"""File defining the API itself, using fastAPI and schemas, and calling the cruds functions"""
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.core.tags import Tags
+from app.cruds import cruds_groups
+from app.dependencies import get_db
+from app.schemas import schemas_core
+
+router = APIRouter()
+
+
+@router.get(
+    "/groups/",
+    response_model=list[schemas_core.CoreGroupSimple],
+    status_code=200,
+    tags=[Tags.groups],
+)
+async def get_groups(db: AsyncSession = Depends(get_db)):
+    """Return all groups from database as a list of dictionaries"""
```

Comment:
The returned object is more a list of `schemas_core.CoreGroupSimple` than a dictionary.

```suggestion
    """Return all groups"""
```